scripts/build.py

```diff
@@ -818,6 +826,19 @@ def _findjslibs(libdir):
             paths = _findjslibs(jsdir)
             self._add_files_to_zip(zip, paths, self.dynamo_home, topfolder)
 
+            # .proto files
+            protodir = os.path.join(self.dynamo_home, 'share/proto/')
+            paths = _findfiles(protodir, ('.proto',))
+            self._add_files_to_zip(zip, paths, self.dynamo_home, topfolder)
+
+            # protoc
+            if platform in ('x86_64-darwin','x86_64-linux','x86_64-win32'): # needed for the linux build server
```

Comment:
I include the darwin/win32 versions here too, for our users to be able to write tools with them.